funnel/templates/proposal_admin_panel.html.jinja2

```diff
@@ -2,87 +2,43 @@
 {% from "baseframe/components.html.jinja2" import faicon %}
 {%- from "macros.html.jinja2" import csrf_tag %}
 
-<div>
-  <p><a class="modal__close mui--text-dark" href="javascript:void(0);" aria-label="{% trans %}Close{% endtrans %}" rel="modal:close" data-cy="close-admin-panel">{{ faicon(icon='times', icon_size='title') }}</a></p>
-  {%- if 'edit_proposal' in proposal.permissions(current_auth.user) or proposal.current_roles.project_editor %}
-    <div class="card">
-      <div class="card__header">
-        <h3 class="mui--text-title mui--text-bold mui--text-uppercase" data-cy="admin-panel">{% trans %}Admin panel{% endtrans %}</h3>
+<div class="modal__header">
+  <a class="modal__close mui--text-dark" href="javascript:void(0);" aria-label="{% trans %}Close{% endtrans %}" rel="modal:close" data-cy="close-admin-panel">{{ faicon(icon='times', icon_size='title') }}</a></p>
+  <h3 class="mui--text-title mui--text-bold mui--text-uppercase" data-cy="admin-panel">{% trans %}Admin panel{% endtrans %}</h3>
```

Comment:
Replace "Admin panel" with "Editor panel" as we have Editor role in projects and Admin role in profiles, which is not extended to projects.